pages/testimonylist.tsx

```diff
@@ -0,0 +1,31 @@
+import { createPage } from "../components/page"
+import { Form, Row, Button } from "../components/bootstrap"
+import Testimonies from "../components/Testimonies/Testimonies"
+
+export default createPage({
```

Comment:
I wonder if this route would read a bit better as /testimonies. Like /bills for the bill list and /bill for a specific bill

Reply:
Matt improvised the name /testimonylist but /testimonies is probably better